File: koin/src/main/java/in/koreatech/koin/data/network/response/UserInfoEditResponse.java
package in.koreatech.koin.data.network.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class UserInfoEditResponse {

    @SerializedName("success")
    @Expose
    public String success;

    @SerializedName("error")
    @Expose
    public String error;


    @SerializedName("message")
    @Expose
    public String message;

}

File: data/src/main/java/in/koreatech/koin/data/response/user/UserInfoEditResponse.kt
package `in`.koreatech.koin.data.response.user

import com.google.gson.annotations.SerializedName

data class UserInfoEditResponse(
    @SerializedName("success") val success: String,
    @SerializedName("error") val error: String,
    @SerializedName("message") val message: String
)
